Enterprise-CMCS/macpro-mdct-mcr
# Remove unused code and dependencies in uploads

### Description

- Remove all unreferenced dependencies
- Upgrade `serverless-s3-local` to latest, removing old aws-sdk 2 sub-dependency
- Remove unused api, scan, and test.json files

---
### How to test

- Virus scan works in deployed env: https://d3g5rqdomzar9t.cloudfront.net/
- Template downloads still work in the deployed env
- No new lambda errors in upload lambdas
  - I tested with our templates and with empty test files. All came back clean, no errors
    - You can check the `uploads-clean-uploads-attachments-*` bucket in AWS to verify the files are tagged clean
    - You can check the `uploads-clean-uploads-avscan` lambda logs to verify everything works
- Local run works
